atomic_reactor/schemas/config.json

```diff
@@ -652,11 +652,19 @@
             "allOf": [{"$ref": "#/definitions/media_types"}]
           },
           "blacklist_srpms": {
-              "description": "Don't include specified srpms to source container image, specify SRPM name",
-              "type": "array",
```

Comment:
Should we keep accepting the array of SRPM names as an option to keep backwards compatibility? Not sure if this would be relevant here though.

Reply:
not really, it is just configuration change, which we will have to do anyway